tekton/publish.yaml

```diff
@@ -61,7 +61,7 @@ spec:
   steps:
 
   - name: container-registy-auth
-    image: gcr.io/go-containerregistry/crane:debug
+    image: cgr.dev/chainguard/crane:latest-dev
```

Comment:
We don't pin crane today, which is probably ok, even though this means that new crane releases might break the release process

Reply:
Done.